com.microsoft.mrtk.input/Editor/Inspectors/ControllerSimulationSettingsDrawer.cs

```diff
@@ -17,7 +17,10 @@ public class ControllerSimulationSettingsDrawer : PropertyDrawer
 
         private readonly GUIContent anchorPointContent = new GUIContent("Anchor point");
         private readonly GUIContent defaultPositionContent = new GUIContent("Default position");
-        private readonly GUIContent defaultPoseContent = new GUIContent("Default pose");
+        private readonly GUIContent defaultGestureContent = new GUIContent("Default gesture");
+        private readonly GUIContent TriggerGestureContent = new GUIContent("Trigger gesture");
+        private readonly GUIContent secondaryDefaultGestureContent = new GUIContent("Secondary default gesture");
+        private readonly GUIContent secondaryTriggerGestureContent = new GUIContent("Secondary trigger gesture");
```

Comment:
nit: inconsistent capitalization of `TriggerGestureContent` compared to the rest of the fields